lib/proto.js

```diff
@@ -91,13 +92,26 @@ function child (bindings) {
       instance[serializersSym][bks] = bindings.serializers[bks]
     }
   } else instance[serializersSym] = serializers
+  if (bindings.hasOwnProperty('formatters')) {
+    instance[formattersSym] = buildFormatters(
+      bindings.formatters.level || formatters.level,
+      bindings.formatters.bindings || resetChildingsFormatter,
+      bindings.formatters.log || formatters.log
```

Comment:
please extract the `bindings.formatters` object into a local `const`, it will shelve some ms.